nwp/jobs.py

```diff
@@ -21,31 +25,42 @@ def build_config_on_runtime(model, run, delay=0):
 for r in ["00", "06", "12", "18"]:
     for model in ["global", "eu"]:
         for delay in [0, 1]:
-            asset_job = define_asset_job(f"download_{model}_run_{r}_{'today' if delay == 0 else 'yesterday'}", AssetSelection.all(),
-                                         config={
-                                             'ops': {"download_model_files": {"config": build_config_on_runtime(model, r, delay)},
-                                                     "process_model_files": {"config": build_config_on_runtime(model, r, delay)},
-                                                     "upload_model_files_to_hf": {
-                                                         "config": build_config_on_runtime(model, r, delay)}, }})
-            if delay == 0:
-                if r == "00":
-                    schedule = ScheduleDefinition(job=asset_job, cron_schedule="30 4 * * *")
-                elif r == "06":
-                    schedule = ScheduleDefinition(job=asset_job, cron_schedule="30 10 * * *")
-                elif r == "12":
-                    schedule = ScheduleDefinition(job=asset_job, cron_schedule="30 16 * * *")
-                elif r == "18":
-                    schedule = ScheduleDefinition(job=asset_job, cron_schedule="30 22 * * *")
-            elif delay == 1:
-                if r == "00":
-                    schedule = ScheduleDefinition(job=asset_job, cron_schedule="1 0 * * *")
-                elif r == "06":
-                    schedule = ScheduleDefinition(job=asset_job, cron_schedule="0 2 * * *")
-                elif r == "12":
-                    schedule = ScheduleDefinition(job=asset_job, cron_schedule="0 6 * * *")
-                elif r == "18":
-                    schedule = ScheduleDefinition(job=asset_job, cron_schedule="0 8 * * *")
-
+            asset_job = define_asset_job(
+                name=f"download_{model}_run_{r}_{'today' if delay == 0 else 'yesterday'}",
+                selection=AssetSelection.all(),
```

Comment:
I think this needs to be changed, but from the docs, I'm not really sure how? In the UI on leonardo, I was seeing all the jobs, including the ECMWF ones, and satellite one,  show up in the assets to be materialized. Not sure how to select the assets based off the prefix, or if the prefix would just be `icon` here.

Reply:
Potentially you're seeing nwp and sat in the UI on leonardo because I specified in the `dagster.yaml` (`/home/dagster/dagster.yaml`) file to display `Definitions` from  both the `sat` and `nwp` modules, since we'll only have one UI to look at on leonardo - not sure if that's what you're referring to here?